### DISCUSSION TITLE
PageUtilities in MAUI

### DISCUSSION THREAD

**Discussion by AngeloLazzari** [PrismLibrary/Prism#3216] [.NET MAUI] [answered]
Hi, we are migrating from Xamarin to MAUI our app. 
We have an error that PageUtilities is not doesn't exist in MAUI scope, and if we check the old xamarin app it was poiting to Prism.Common... is that still there? 

We are experiencing this issue https://developercommunity.visualstudio.com/t/Second-time-reporting:-WPF:-C-Editor-ca/10532548 on our visual studio too, and probably the error comes from there, but I ask you just to be sure... 

thank you very much

**Comment by dansiegel** [comment, marked as answer]
PageUtilities was misnamed in Prism.Forms. As part of the move to better unify the API's this was renamed MvvmUtilties in Prism.Maui.

https://github.com/PrismLibrary/Prism/blob/master/src/Maui/Prism.Maui/Common/MvvmHelpers.cs

**Reply by AngeloLazzari** [reply]
ok, thanks, I changed it.
But is there a doc where I can check all that kind of changes you did? So we can follow it? Thank you very much!
